jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -564,7 +564,7 @@ def compute_scale(wcs, fiducial, disp_axis, pscale_ratio=1):
     """
     spectral = 'SPECTRAL' in wcs.output_frame.axes_type
 
-    if spectral and disp_axis is None:
+    if spectral and disp_axis is None:  # pragma: no cover

```

Comment:
NOTE: This was copied from `jwst` so `if spectral` is probably important there but we do not care here.